<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSAM</class>
 <widget class="QDialog" name="DlgSAM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Спектральная угловая корреляция</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="1">
      <widget class="QPushButton" name="ButtonDell">
       <property name="text">
        <string>Убрать&gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="ButtonOK">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="ButtonCancel">
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="ButtonAdd">
       <property name="text">
        <string>&lt;&lt;Добавить</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QRadioButton" name="RadioSKOEnable">
       <property name="text">
        <string>С учётом СКО угловой меры</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Ограничение стандартного отклонения (град: 0-180)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="RadioSKODisable">
       <property name="text">
        <string>Без учёта отклонения угловой меры</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="EditSKOLimit"/>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="3">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Учитываемые при классификации каналы:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="ListCh"/>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Эталонные полигоны</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="ListEtalon"/>
       </item>
      </layout>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
